kod/object/active/holder/nomoveon/battler/monster/towns/barlqtwn/bqinnk.kod

```diff
@@ -27,18 +27,21 @@ resources:
    BqInn_mace_sale = "Aye, ye might needeth one of them if ye plan on leavingeth our cultured city. I got some ye can buyeth."
    BqInn_unwanted_give = "~k%s%s tells you, \"Whyfore dost you offer me that?~n~k\""
 
-   BarloqueInnkeeper_room_rented = "Here is the key to thine room.  I trust you will findest "
+   BarloqueInnkeeper_room_rented = "Here is the key to thy room.  I trust you will findest "
```

Comment:
Mayhem made the point that maybe these "thine"s were wrong intentionally?  If so I'm OK with it.  Either way.

Reply:
I reverted the fixes (thy back to thine, as well as a thee back to thou from earlier) and left a note so that it hopefully won't cause any confusion to anyone else going forward.